src/java.base/share/classes/java/io/ObjectInputStream.java

```diff
@@ -2013,6 +2011,13 @@ private ObjectStreamClass readProxyDesc(boolean unshared)
         return desc;
     }
 
+    // Generate an InvalidObjectException for an OutOfMemoryError
+    // Use String.concat() to avoid string formatting invoke dynamic
+    private static InvalidObjectException genInvalidObjectException(OutOfMemoryError oome, String[] ifaces) {
```

Comment:
Isn't this new line is too long? It seems most of the file uses 80 chars per line.